src/thread.rs

```diff
@@ -96,7 +96,8 @@ mod test {
             let time_before = time::SteadyTime::now();
             {
                 let _raii_joiner = RaiiThreadJoiner::new(thread!("JoinerTestManaged", move || {
-                    ::std::thread::sleep(::std::time::Duration::from_millis(SLEEP_DURATION_MANAGED));
+                    ::std::thread::sleep(
```

Comment:
These still have fully qualified namespaces - So i think the rust rules (in their style guide) is that types be completely unqualified (so `use std::time::Duration` etc.) while the functions be qualified by the final module they are in (so `use std::thread` and `thread::sleep(...)`)